client/components/loading-placeholder/index.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import noop from 'lodash/utility/noop';
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import Main from 'components/main';
+
+const LoadingPlaceholder = React.createClass( {
```

Comment:
I see your point but still I wouldn't suffix this component's name with `Page` for consistency purpose. We could name it `<Skeleton />` instead since this is how we've been referring to it for some time now. I'm fine with renaming any placeholder element inside it as `<Bone />` though :).